### PR TITLE
[FEAT] Add implementation of connection handler

### PR DESCRIPTION
**Pull request by LeaYeh**
@yenthing check the webshell interface and naming convention in commit https://github.com/LeaYeh/webserver/pull/40/commits/e9f8efc9d9152d7839ccfbb9b92622b5f233cafb

### CODE REVIEW

**Review by yenthing — COMMENTED** [review]
1. In define.hpp there are some typo. 
2. For function "handle_event" in Acceptor.cpp, why did you change EPOLLOUT flag into EPOLLET flag?

**Comment by yenthing** [comment]
Can you please add Chore into [verb] for commit message, because its in the collaborate docs but not in the test. Upsi

**Comment by yenthing** [comment]
> Can you please add Chore into [verb] for commit message, because its in the collaborate docs but not in the test. Upsi

[FIXED] We did it! Muahahahaha

File: source/kernel/Acceptor.cpp [hunk 1]
```diff
@@ -52,12 +60,17 @@ void Acceptor::handle_event(int fd, uint32_t events)
             throw std::runtime_error("accept() failed: " +
                                      std::string(strerror(errno)));
         ConnectionHandler* conn_handler =
-            new ConnectionHandler(conn_fd, _reactor);
-        _reactor->register_handler(conn_fd, conn_handler, EPOLLIN | EPOLLOUT);
+            new ConnectionHandler(conn_fd, _reactor, _config, _server_id);
+        _reactor->register_handler(conn_fd, conn_handler, EPOLLIN | EPOLLET);
```
Comment (LeaYeh) [inline source/kernel/Acceptor.cpp:64]: The EPOLLOUT is for output event but the EPOLLET is for edge trigger enevts

https://github.com/LeaYeh/webserver/blob/main/doc/non-blocking%20IO.md#epoll

Comment (LeaYeh) [reply source/kernel/Acceptor.cpp:64]: tbh I'm not sure can I remove EPOLLIN or not

**Review by leske42 — APPROVED** [review]

**Review by yenthing — APPROVED** [review]

**Pull request merged**